src/app/templates/index.ftl

```diff
@@ -156,9 +156,11 @@
                 </#list>];
             </#if>
             var runReportBtn = document.querySelector('#run-report')
-            runReportBtn.addEventListener('click', () => {
-                window.sessionStorage.removeItem("selectedRunningReportTab");
-            })
+            if (runReportBtn) {
```

Comment:
Just trying to quieten the error output - the old code threw a js error if the user didn't have run report permissions